### ISSUE TITLE
Dynamic Report block changes Is Equal to Contains in Data View

### ISSUE THREAD

**Issue by cabal95** [SparkDevNetwork/Rock#4242]
### Description
When using the Dynamic Report block, it modifies the original Data View filter criteria from `Is Equal` to become `Contains`. I chose `Is Equal` because I specifically wanted it to be Equal. If I wanted it give users the ability to do a sub string search I would specifically choose Contains.

### Steps to Reproduce
1. Go to demo site.
1. Create a new Person Data View with a filter of `Last Name Is Equal To Decker`
1. Create a new Person Report and put the Person Name field on.
1. Add a Dynamic Report block to a page and configure it to use the report you just created.
1. BEFORE YOU SAVE, notice that the filter is seen as a "Equal To" type of filter.
1. Click Save.
1. Edit block settings again.
1. Notice the filter is now shown as a "Contains" type of filter.
1. Go back to your Data View and notice it has actually changed the type of the search in the Data View.
1. Change it back to "Equal To" and save Data View.
1. Go back to the dynamic report and reload the page.
1. Go look at the Data View and notice it again changed it back to "Contains".
1. Claw your face in frustration.
1. Buy some make up to cover all those scratches.

**Expected behavior:**

The Dynamic Report block should only be changing the specific _content_ of what is being searched for, not the type of search that is being performed.

**Actual behavior:**

Your carefully crafted Data View is suddenly not doing what you told users it would do.

### Versions
* **Rock Version:** 10.2, 10.3-alpha
* **Client Culture Setting:** en-US

┆Attachments: <a href="https://app.asana.com/app/asana/-/get_asset?asset_id=1202009384069175">image.png</a> | <a href="https://app.asana.com/app/asana/-/get_asset?asset_id=1202009384069177">image.png</a>

**Comment by chead4** [comment]
The Dynamic Report block should be getting converted to Obsidian fairly soon, as we expect that to fix this issue we will close it for now. If you still encounter this issue once that conversion is complete please re-open or file a new issue.

**Comment by dataCollegechurch** [comment]
We ran into this bug today. I created a dynamic report staff can use to search for people by Membership Number. Membership Number is a person attribute in our system. This type of search is helpful as names can change over-time but membership numbers do not. We are in the process of digitizing old membership applications. Each application does have a membership id written on it so even if you cannot find the person by name, you may be able to locate the correct record by membership id. We have thousands of members so searching for someone with the membership number 7 will return hundreds of results as hundreds of membership ids contain the number 7.

**Comment by chead4** [comment]
@dataCollegechurch Thank you for reaching out.  You're correct, the Dynamic Report block is not available yet in obsidian.
